dfttopif/parsers/wien2k.py

```diff
@@ -0,0 +1,174 @@
+from pypif.obj import *
+from .base import DFTParser, Value_if_true
+import os
+from dftparse.util import *
+from dftparse.wien2k.scf_parser import ScfParser
+from dftparse.wien2k.scf2_parser import Scf2Parser
+from dftparse.wien2k.absorp_parser import AbsorpParser
+
+
+class Wien2kParser(DFTParser):
+    """
+    Parser for Wien2k calculations
+    """
+
+    def get_name(self): return "Wien2k"
+    
+    def test_if_from(self, directory):
+        # Check whether it has a .scf file (analogous to an OUTCAR file)
+        for filename in os.listdir(directory):
+            if os.path.splitext(filename)[1] == ".scf":
```

Comment:
This could match some pwscf files.  Could you inspect the `scf` file for some `wien2k`-only info?